data-structures/src/data/structure/linkedlist/LinkedList.java

```diff
@@ -0,0 +1,239 @@
+package data.structure.linkedlist;
+import static  java.util.Objects.*;

```

Comment:
Please use individual imports instead of the star import. It makes for more longer but more straightforward code. Also, there are two spaces between `static` and `java` :D